contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/index.vue

```diff
@@ -376,7 +454,10 @@
         () => currentChannel.value?.publishing,
```

Comment:
I did not flag this earlier because I wanted to have the complete picture regarding this `currentChannel` prop with this PR. And now that it seems that `props.channel` is indeed reactive, then I think we can just always use `props.channel`, and avoid the definition of this `currentChannel` [here](https://github.com/taoerman/studio/blob/27fd389f3aff570ea213a1d08e104b6ba102acd0/contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/index.vue#L244)?

The pros of using `props.channel` instead of `currentChannel` are that we make this side panel less coupled to the channel edit context. And if, for some reason, we want to show it in another place, it will be easier to implement.